docs/data/data-grid/filtering/CustomRatingOperator.tsx

```diff
@@ -4,12 +4,19 @@ import Rating, { RatingProps } from '@mui/material/Rating';
 import {
   GridFilterInputValueProps,
   DataGrid,
-  GridFilterItem,
   GridFilterOperator,
   GridToolbarFilterButton,
 } from '@mui/x-data-grid';
 import { useDemoData } from '@mui/x-data-grid-generator';
 
+function Toolbar() {
+  return (
+    <div>
+      <GridToolbarFilterButton />
+    </div>
+  );
+}
+
```

Comment:
Curious, why do we wrap in an additional `div` now?

Reply:
I didn't like the filter button expanding on full width here 😅
https://mui.com/x/react-data-grid/filtering/customization/

<img width="815" src="https://github.com/mui/mui-x/assets/13808724/aeec51c5-8e64-4535-bba9-9314f8129cf1">